roles/ceph-osd/tasks/start_osds.yml

```diff
@@ -32,5 +34,6 @@
   service:
     name: ceph-osd@{{ item }}
     state: started
+    enabled: true
```

Comment:
That's not true, we also call `ceph-disk activate`. Anyway, I understand the issue, so if you believe that's the right approach then I don't have any objection to this patch.